### ISSUE TITLE
RCS0056 with raw strings issue

### ISSUE THREAD

**Issue by prezaei** [dotnet/roslynator#1637]
The following block, triggers RCS0056:

```csharp
        var a = $$"""
            - “Hey, {{userContext.User.FirstName}}! test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test .”
            hey line 2.
            """;
```

This also triggers it:

```csharp
        var a = $$"""
            - “Hey, {{ "name" }} test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test .”
            hey line 2.
            """;
```

but this doesn't:

```csharp
        var a = """
            - “Hey, test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test .”
            hey line 2.
            """;
```
